.github/workflows/remove-lockdown-label.yml

```diff
@@ -0,0 +1,74 @@
+        steps:
+        -   name: Checkout repository
+            uses: actions/checkout@v2
+
+        -   name: PR's only change is <VersionFeature> in eng/Versions.props
+            id: only_version_feature_changed
+            uses: actions/github-script@v4
+            with:
+                script: |
+                    const otherChangesMessage = "❌ Changes in eng/Versions.props other than <VersionFeature> found";
+                    const onlyVersionFeatureMessage = "✅ PR's only change is <VersionFeature> in eng/Versions.props";
+                    const prNumber = context.payload.pull_request.number;
+                    const { data: files } = await github.pulls.listFiles({
+                        owner: context.repo.owner,
+                        repo: context.repo.repo,
+                        pull_number: prNumber
+                    });
+                    // If files other than eng/Versions.props are changed, output message and exit
+                    if (files.some(file => file.filename !== "eng/Versions.props")) {
+                        console.log(otherChangesMessage);
+                        core.exportVariable("only_version_feature_changed", "false");
+                        return;
+                    }
+                    // Iterate through the patch of eng/Versions.props to check for changes other than <VersionFeature>
+                    const versionsPropsFile = files.find(file => file.filename === "eng/Versions.props");
+                    const patchLines = versionsPropsFile.patch.split("\n").filter(l => l.startsWith("+") || l.startsWith("-"));
+                    for (const line of patchLines) {
+                        if (!line.includes("<VersionFeature>")) {
+                            console.log(otherChangesMessage);
+                            core.exportVariable("only_version_feature_changed", "false");
+                            return;
+                        }
+                    }
+                    console.log(onlyVersionFeatureMessage);
+                    core.exportVariable("only_version_feature_changed", "true");
+
+        -   name: Remove Branch Lockdown label from other PRs targeting this branch
+            if: steps.only_version_feature_changed.outputs.only_version_feature_changed == 'true'
+            uses: actions/github-script@v4
+            with:
+                script: |
+                    const prs = await github.pulls.list({
+                        owner: context.repo.owner,
+                        repo: context.repo.repo,
+                        state: 'open',
+                        base: github.event.pull_request.base.ref,
+                        per_page: 300
+                    });
+                    const filtered_prs = prs.data
+                        .filter(pr => pr.number !== github.context.payload.pull_request.number)
+                        .filter(pr => pr.labels.map(label => label.name).includes('Branch Lockdown'));
```

Comment:
equals instead of includes?

Reply:
I think `pr.labels` is an array

Reply:
Oh, I see. I saw the map and started thinking about it as a string, but you just mapped label to its name. Makes sense.